server/log_rpc_server_test.go

```diff
@@ -1146,178 +1146,167 @@ func TestGetConsistencyProofBeginTXFails(t *testing.T) {
-			t.EXPECT().ReadRevision().Return(signedRoot1.TreeRevision)
-			t.EXPECT().GetMerkleNodes(gomock.Any(), revision1, nodeIdsConsistencySize4ToSize7).Return([]storage.Node{}, errors.New("STORAGE"))
-		},
-		func(s *TrillianLogRPCServer) error {
-			_, err := s.GetConsistencyProof(context.Background(), &getConsistencyProofRequest7)
-			return err
-		})
-
-	test.executeStorageFailureTest(t, getConsistencyProofRequest7.LogId)
-}
-
-func TestGetConsistencyProofGetNodesReturnsWrongCount(t *testing.T) {
-	ctrl := gomock.NewController(t)
-	defer ctrl.Finish()
-
-	mockStorage := storage.NewMockLogStorage(ctrl)
-	mockTx := storage.NewMockLogTreeTX(ctrl)
-	mockStorage.EXPECT().SnapshotForTree(gomock.Any(), getConsistencyProofRequest7.LogId).Return(mockTx, nil)
-
-	mockTx.EXPECT().LatestSignedLogRoot(gomock.Any()).Return(signedRoot1, nil)
-	mockTx.EXPECT().ReadRevision().Return(signedRoot1.TreeRevision)
-	// The server expects one node from storage but we return two
-	mockTx.EXPECT().GetMerkleNodes(gomock.Any(), revision1, nodeIdsConsistencySize4ToSize7).Return([]storage.Node{{NodeRevision: 3}, {NodeRevision: 2}}, nil)
-	mockTx.EXPECT().Close().Return(nil)
-
-	registry := extension.Registry{
-		AdminStorage: mockAdminStorage(ctrl, getConsistencyProofRequest7.LogId),
-		LogStorage:   mockStorage,
-	}
-	server := NewTrillianLogRPCServer(registry, fakeTimeSource)
-
-	_, err := server.GetConsistencyProof(context.Background(), &getConsistencyProofRequest7)
-	if err == nil || !strings.Contains(err.Error(), "expected 1 nodes") {
-		t.Fatalf("get consistency proof returned no or wrong error when get nodes returns wrong count: %v", err)
-	}
-}
-
-func TestGetConsistencyProofGetNodesReturnsWrongNode(t *testing.T) {
-	ctrl := gomock.NewController(t)
-	defer ctrl.Finish()
-
-	mockStorage := storage.NewMockLogStorage(ctrl)
-	mockTx := storage.NewMockLogTreeTX(ctrl)
-	mockStorage.EXPECT().SnapshotForTree(gomock.Any(), getConsistencyProofRequest7.LogId).Return(mockTx, nil)
-
-	mockTx.EXPECT().LatestSignedLogRoot(gomock.Any()).Return(signedRoot1, nil)
-	mockTx.EXPECT().ReadRevision().Return(signedRoot1.TreeRevision)
-	// Return an unexpected node that wasn't requested
-	mockTx.EXPECT().GetMerkleNodes(gomock.Any(), revision1, nodeIdsConsistencySize4ToSize7).Return([]storage.Node{{NodeID: stestonly.MustCreateNodeIDForTreeCoords(1, 2, 64), NodeRevision: 3}}, nil)
-	mockTx.EXPECT().Close().Return(nil)
-
-	registry := extension.Registry{
-		AdminStorage: mockAdminStorage(ctrl, getConsistencyProofRequest7.LogId),
-		LogStorage:   mockStorage,
-	}
-	server := NewTrillianLogRPCServer(registry, fakeTimeSource)
-
-	_, err := server.GetConsistencyProof(context.Background(), &getConsistencyProofRequest7)
-	if err == nil || !strings.Contains(err.Error(), "expected node ") {
-		t.Fatalf("get consistency proof returned no or wrong error when get nodes returns wrong node: %v", err)
-	}
+type consistProofTest struct {
```

Comment:
As this is a big struct, it might be nice to order the fields in groups:
 - overall input (`req`)
 - overall expected outputs (`wantErr`, `hashes`)
 - expected mock operations along the way, in chronological order:
     - snapshot (`noSnap`, `snapErr`)
     - get root (`noRoot`, `rootErr`)
     - get revision (`noRev`)
     - get nodes (`nodeIDs`, `nodes`, `getNodesErr`)
     - commit (`noCommit`, `commitErr`)
     - close (`noClose`)

and make the test cases use the same order throughout.

Reply:
Done. I was using a different grouping  by function.